src/test/java/eu/yals/test/utils/elements/YalsElement.java

```diff
@@ -5,89 +5,89 @@
 import org.openqa.selenium.WebElement;
 
 /**
- * Test methods for Selenide Elements
+ * Test methods for Selenide Elements.
  *
  * @since 2.7
  */
 public class YalsElement {
-  private final WebElement webElement;
+    private final WebElement webElement;
 
-  public static YalsElement wrap(WebElement element) {
-    return new YalsElement(element);
-  }
+    public static YalsElement wrap(WebElement element) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter element should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)